scripts/screeps_rl_simulator_harness.py

```
@@ -1631,6 +1656,51 @@ def collect_runtime_parameter_consumption_evidence(
     return None, errors
 
 
+def collect_runtime_parameter_consumption_evidence_with_retries(
+    smoke: Any,
```

Ruff (0.15.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `smoke`

(ANN401)